draft-ietf-quic-transport.md

```diff
@@ -1257,22 +1257,21 @@ way to retry them.
 
 ### Privacy Implications of Connection Migration {#migration-linkability}
 
-Using a stable connection ID on multiple network paths allows a passive observer
-to correlate activity between those paths.  A client that moves between networks
-might not wish to have their activity correlated by any entity other than a
-server.
-
-The NEW_CONNECTION_ID message can be sent by a server to provide an unlinkable
-connection ID for use in case the client wishes to explicitly break linkability
+Using a stable connection ID on multiple network paths allows a
+passive observer to correlate activity between those paths.  A client
+that moves between networks might not wish to have their activity
+correlated by any entity other than a server. The NEW_CONNECTION_ID
+message can be sent by a server to provide an unlinkable connection ID
+for use in case the client wishes to explicitly break linkability
 between two points of network attachment.
 
-The NEW_CONNECTION_ID message includes a connection ID that the client can use
-for packets on the current connection.  This message also includes packet number
-gap, which is the number of packet numbers that the client skips when using the
-new connection ID.  Without this, the packet number field in the header might be
-used to link activity across networks.  A server MUST provide a 32-bit value for
-the packet number gap field that is indistinguishable from random to a passive
-observer.
+A client which wishes to break linkability upon changing networks MUST
+use the NEW_CONNECTION_ID as well as incrementing the packet sequence
```

Comment:
`MUST use the NEW_CONNECTION_ID` implies sending, it's the value from the message that the client uses.

I'm not sure about the MUST-requirement.  Isn't this a simple "can"?